pom.xml

```diff
@@ -8,7 +8,7 @@
   <description>Collection of non-Vaadin, non-Liferay utilities.</description>
   <modelVersion>4.0.0</modelVersion>
   <name>Core Utilities Library</name>
-  <version>1.8.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
+  <version>1.9.0-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
```

Comment:
The SNAPSHOT Tag will be removed automatically right?

Reply:
yes exactly